app/helpers/avatar_helper.rb

```diff
@@ -28,7 +28,7 @@ def avatar_image(user, options = {})
       alt: alt,
       data: { fallback_image: fallback_image },
       height: size,
-      style: "width: #{size}px; height: #{size}px",
+      style: "width: #{size}px; height: #{size}px; border-radius: 50%;",
```

Comment:
this doesn't belong here? the spirit of this `:style` was to include sizing information only, the webapp already has the `border-radius` CSS elsewhere.

I understand this is here to make it look nice in the embedded mail views, however we may need to reconsider passing a `:style` key to the options method that gets merged here. That way the mailer view can use it, and we avoid repeating this inline redundant rule all over the app?

Reply:
✅